packages/rum-core/src/domain/foregroundContexts.ts

```diff
@@ -60,10 +51,6 @@ function addNewForegroundPeriod() {
   }
   const currentForegroundPeriod = foregroundPeriods[foregroundPeriods.length - 1]
   if (currentForegroundPeriod !== undefined && currentForegroundPeriod.end === undefined) {
-    addMonitoringMessage('Previous foreground periods not closed. Continuing current one', {
```

Comment:
What about other messages?

Reply:
AFAIK, They are not triggered at the moment, I would like to know if we change something and those limit case start to trigger.

Reply:
By looking at the logs I shared with you earlier, we can see some occurrences for all of them.